medicines/api/README.md

```diff
@@ -43,25 +43,38 @@ To see the graphql explorer, go to http://127.0.0.1:8080/graphiql.
 ## Deploy API pod in Kuberbetes cluster ⎈
 
 1. Navigate to this directory, `/medicines/api`
-2. Create [Kubernetes Service][kubernetes service]
+2. Source environmental variables from the correspondent environment
+   ```sh
+   source ./infrastructure/non-prod/.env
+   ```
+3. Create [Kubernetes secret][kubernetes secret] for API Pod to consume
```

Comment:
will this step need to be repeated each time we deploy an API pod into the cluster?

Reply:
Nope

Reply:
What does someone do if they want to check whether this step needs to be done?

Reply:
Run the command and if they are created it would notify that